perf/CMakeLists.txt

```diff
@@ -76,6 +78,13 @@ function(create_perf_test_target)
 
   set(RUN_PERF_C_TESTS_LIST ${RUN_PERF_C_TESTS_LIST} ${BENCH_TARGET} PARENT_SCOPE)
   set(BENCH_RESULTS ${BENCH_RESULT_TARGET} ${BENCH_RESULTS} PARENT_SCOPE)
+  set(test_title "perf/${PERF_TARGET}.cc")
+  add_test(NAME ${test_title}
+           COMMAND "$<TARGET_FILE:${PERF_TARGET}.perftest>")
+  set_tests_properties(${test_title} PROPERTIES
+    LABELS "${TEST_SUITE_NAME};performance;long"
```

Comment:
I propose dropping the "long" label to avoid clashing with long tests.

Reply:
Performance test are long as well, where is a clash? Imagine you want to run "fast" test only and exclude "long" tests. Then just execute `ctest -LE long`.

Reply:
OK, lets leave as is for now.